config/remote_config.go

```diff
@@ -40,7 +40,7 @@ type RemoteConfig struct {
 	TimeoutStr string            `default:"5s" yaml:"timeout" json:"timeout,omitempty"`
 	Username   string            `yaml:"username" json:"username,omitempty" property:"username"`
 	Password   string            `yaml:"password" json:"password,omitempty"  property:"password"`
-	Params     map[string]string `yaml:"params" json:"address,omitempty"`
+	Params     map[string]string `yaml:"params" json:"params,omitempty"`
```

Comment:
pls do not change the json tag.

Reply:
https://github.com/apache/dubbo-go/blob/811f4f1a2dcc75b4294f62cb795da0f3f399be93/config/remote_config.go#L37-L44

This is another json tag `address` for `Address`. I think that tag `address` for `Params` is duplicated and wrong.

Reply:
ok. u r right.